Story/Episode 3/un_035010.csv

```diff
@@ -0,0 +1,35 @@
+name01#0,"""???"""
+name02#0,"""???"""
+un_035010_0010#0,"""<%me>!<br>I wanted to see you again!"""
+un_035010_0020#0,"""<%me>, listen!<br>Sir Saga has been terrible with me!"""
+un_035010_0030#0,"""Putting me through such intense training without any rest, my body has been pushed to its limit."""
+un_035010_0040#0,"""...Yet you do have the stamina to go running to greet <%me>."""
+un_035010_0050#0,"""That was but a sudden burst. Aah, my body's starting to feel fragile again..."""
+un_035010_0060#0,"""Enough. You are annoying <%me>."""
+un_035010_0070#0,"""So harsh! Unemotional as usual!"""
+un_035010_0080#0,"""Aah~ If only I could be partnered up with <%me> instead of Saga."""
+un_035010_0090#0,"""...I understand. If that's what it takes, it can't be helped. Would you rather try it that way?"""
+un_035010_0100#0,"""...Eh?"""
+un_035010_0110#0,"""I've reached the limits of what I can teach, I'm afraid."""
+un_035010_0120#0,"""I'll take care of the paperwork. Your training will remain the same, but you'll have a new partner."""
+un_035010_0130#0,"""Ah? N-no...<br>Sir Saga? What are you..."""
+un_035010_0140#0,"""<%me>, my apologies.<br>From now on, you'll have to take care of it. I'm counting on you."""
+un_035010_0150#0,"""Ho-hold on! Wait! Sir Saga, I was jesting! A joke! I'm sure you understand!"""
+un_035010_0160#0,"""...Don't overstrain yourself, Katori. After all our training, your body's in a lot of pain, isn't it?"""
+un_035010_0170#0,"""I see now that I've been putting you through too much. Forgive me."""
+un_035010_0180#0,"""Such kind words! But how?! They feel like an arrow piercing through my chest!"""
+un_035010_0190#0,"""No, this is wrong! I can do it! I can keep going! Actually, I haven't done enough yet!"""
+un_035010_0200#0,"""...Does that mean you still want me to train you?"""
+un_035010_0210#0,"""But of course!"""
+un_035010_0220#0,"""Even if you feel like you're dying? Even after your body starts crying, you'll keep going?"""
+un_035010_0230#0,"""I'm completely prepared!"""
+un_035010_0240#0,"""You don't really mean those words."""
+un_035010_0250#0,"""I certainly do!"""
+un_035010_0260#0,"""I see..."""
+un_035010_0270#0,"""Then I won't hold back. Our training thus far has only been a warm up. The real thing starts now."""
+un_035010_0280#0,"""With our next training, I'll test your new resolve!"""
+un_035010_0290#0,"""I have already prepared it. Come on, let's go."""
+un_035010_0300#0,"""...Excuse me? Already prepared, you say?"""
+un_035010_0310#0,"""B-but, hold on, Sir Saga. Shouldn't we start taking it easy...?"""
+un_035010_0320#0,"""I knew you didn't mean those words."""
+un_035010_0330#0,"""It is true my morale was high throughout the conversation, b-but...!"""
```

Comment:
Don't like this last line. It's a bit awkward, but I'm not sure how to rewrite it.